src/base/bittorrent/torrenthandle.h

```diff
@@ -82,6 +82,20 @@ namespace libtorrent
 
 namespace BitTorrent
 {
+    namespace prio
```

Comment:
This namespace is redundant. It was a very outdated piece of code...
Either declare FilePriority as enum class or move it to TorrentHandle and use more meaningful names: IgnoredFile, NormalPriorityFile, etc. The second way will be easier for you, because it will require less changes to the existing code (enum class will require explicit cast to int).

Reply:
What's wrong with removing the `prio` namespace and keeping `FilePriority` as an enum (non-class)?

Reply:
Nothing wrong for me if we get meaningful names.

Reply:
To me, ‘BitTorrent::FilePriority::IGNORED’ is descriptive and meaningful, but I take it you don’t agree?

Reply:
Yes, it's descriptive and meaningful. I'm OK with it if you'll use camel case: `BitTorrent::FilePriority::Ignored`.
But the question is, how are you going to get those names? Old style enum doesn't produce its own namespace. So you either have to use the old "perverse" way of declaring it in an extra "FilePriority" namespace, or you still have to use an enum class.

Reply:
I've moved the values into a `FilePriority` class. This also allows us to support an `isValidPriority()` function, which is needed to validate values from the WebAPI.